src/main/java/pl/javastart/task/ArrayUtils.java

```diff
@@ -0,0 +1,28 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class ArrayUtils {
+    public double [] getSize() {
```

Comment:
size to rozmiar, ta metoda nie zwraca rozmiaru, tylko tablicę, więc bardziej pasuje nazwa typu createArray. Dodatkowo nie stawiaj spacji przed nawiasami []